File: rgn_99problems/src/main/java/ragna/HelloWorld.java
package ragna;

import java.util.List;

public class HelloWorld {
    public static void main(String[] args) {
        System.out.println("Hello World!");
        var list = List.of();

        System.out.println(list);
        System.out.println(ALists.head(list));
        System.out.println(ALists.tail(list));

    }

}

File: rgn_99problems/src/main/kotlin/ragna/kt/HelloWorld.kt
package ragna.kt

import kotlinx.collections.immutable.ImmutableList
import kotlinx.collections.immutable.persistentListOf

fun main() {
    println("Hello World KT!")
    val list: ImmutableList<String> = persistentListOf("I", "am", "immutable")
    println(list)
    println(list.dropLast(1))
    println(list)
    println(list.drop(1))
    println(list.first())
}
